ncm-cron/src/main/perl/cron.pm

```diff
@@ -453,76 +511,85 @@ sub cleanCrontabs {
             # Not yet implemented
             next;
         } else { # Linux
-            my $fname = "$name.ncm-cron.cron";;
+            my $fname = "$name$CRON_ENTRY_EXTENSION";;
             $fname =~ s{[/\s]}{_}g;
             $cronEntryNames{$fname} = ''; # Value is meaningless
         }
     }
 
-    # Solaris: Collect the crontabs in /var/spool/cron/crontabs then delete
-    # any ncm-cron entries. These are identified as lines between:
-    #     NCM-CRON START:
-    #     NCM-CRON END:
+    my $crondir = $osname eq "solaris" ? $CRONDIR_SOLARIS : $CRONDIR_LINUX;
+
+    # Files only. See remark about LC::Check::absence below
```

Comment:
ok, but `cleanup` should be modified to support a mode where only files are deleted, and anything else is ignored/skipped (that's what the old `absence(Files => True)` is about